src/lightning_lite/__init__.py

```diff
@@ -1,4 +1,23 @@
 """Root package info."""
+import logging
 
 from lightning_lite.__about__ import *  # noqa: F401, F403
 from lightning_lite.__version__ import version as __version__  # noqa: F401
+
+_root_logger = logging.getLogger()
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+# TODO(lite): Lite and PL interfere through this root_logger, breaking Lite tests who assert through caplog
```

Comment:
This might not be an issue now as each library uses it's own loggers for the rank zero functions